pkg/security/probe/process_resolver.go

```diff
@@ -915,8 +943,14 @@ func NewProcessResolver(probe *Probe, resolvers *Resolvers, client *statsd.Clien
 		argsEnvsCache: argsEnvsCache,
 		state:         snapshotting,
 		argsEnvsPool:  NewArgsEnvsPool(),
+		hitsStats:     map[string]*int64{},
```

Comment:
[minor note] It might be easier to initialize this in-place (`map[string]*int64{metrics.ProcFSTag: 0, ...}`) or initialize it in a local variable with a loop before the `ProcessResolver` creation.  I had a moment of confusion over how the keys for this map get initialized.

Reply:
Yeah I see what you're saying ... unfortunately, the list of tags `metrics.AllTypesTags` might change in the future, and we use it in a few different places in the agent; so initialising the map in-place is risky IMO. I moved up the initialisation loop right below the line you highlighted to hopefully make things clearer 😄